spotinst_sdk2/models/elastigroup/gcp/__init__.py

```diff
@@ -1,124 +1,138 @@
 import json
+from typing import List
 
 none = "d3043820717d74d9a17694c176d39733"
 
 
-# region Elastigroup
-class Elastigroup:
-    """
-    # Arguments
-    name: str
-    description: str
-    capacity: Capacity
-    strategy: Strategy
-    scaling: Scaling
-    third_parties_integration: ThirdPartiesIntegration
-    compute: Compute
-    """
-    def __init__(
-            self,
-            name=none,
-            description=none,
-            capacity=none,
-            strategy=none,
-            scaling=none,
-            third_parties_integration=none,
-            compute=none):
-
-        self.name = name
-        self.description = description
-        self.capacity = capacity
-        self.strategy = strategy
-        self.scaling = scaling
-        self.third_parties_integration = third_parties_integration
-        self.compute = compute
-
-# endregion
-
 # region Capacity
 class Capacity:
     """
     # Arguments
     minimum: int
     maximum: int
     target: int
+    unit: str
     """
-    def __init__(self, minimum=none, maximum=none, target=none):
 
+    def __init__(self,
+                 minimum: int = none,
+                 maximum: int = none,
+                 target: int = none,
+                 unit: str = none):
         self.minimum = minimum
         self.maximum = maximum
         self.target = target
+        self.unit = unit
+
+
 # endregion
 
 
 # region Strategy
+class RevertToPreemptible:
+    """
+    # Arguments
+    perform_at: str
+    """
+
+    def __init__(self,
+                 perform_at: str = none):
+        self.perform_at = perform_at
+
+
 class Strategy:
     """
     # Arguments
     preemptible_percentage: int
     on_demand_count: int
     draining_timeout: int
     fallback_to_od: bool
+    optimization_windows: List[str]
+    provisioning_model: str
```

Comment:
Can we declare provisioning model as enum

Reply:
Done